content/doc/book/pipeline/jenkinsfile.adoc

```diff
@@ -398,6 +399,9 @@ node {
 apply to all steps within the Pipeline.
 <2> An `environment` directive defined within a `stage` will only apply the
 given environment variables to steps within the `stage`.
+<3> An `environment` directive can contain environment variables referring to
```

Comment:
I think this callout could use a brief example, such as:

```asciidoc
<3> An `environment` directive can contain variables referring to other pre-defined environment variables
using either single-quotes (`CFLAGS = '-g $DEBUG_FLAGS'`) or by escaping the dollar-signs when using
double quotes (`CFLAGS = "-g \$DEBUG_FLAGS"`)
```

I'm a bit fuzzy on whether only `${THIS}` syntax is supposed to be supported for references, or like in shells, `$THIS` should work too (with and without curly braces)